scholia/app/templates/chemicals_missing.html

```diff
@@ -0,0 +1,309 @@
+The following links prioritize chemical compounds that are listed in most Wikis
+(Wikipedias, WikiCommons, etc). To go to the Wikidata page for each item, click the name
+below and then on the shown Scholia page for that compound the Q-identifier link at the top.
+
+<h2 id="h1">Missing Boiling Points</h2>
+
+The <a href="https://www.wikidata.org/wiki/Property:P2102">boiling point</a> is the temperature above
+which the compound is liquid. Typical units given include Kelvin and degree Celsius. Because it
+is depends on the pressure, this is often also given, as shown in
+<a href="https://www.wikidata.org/wiki/Property:P2102#P1855">these examples</a>.
+
+<table class="table table-hover" id="missing-boiling-points"></table>
+
+<h2 id="h1">Missing Melting Points</h2>
+
+The <a href="https://www.wikidata.org/wiki/Property:P2101">melting point</a> is the temperature above
+which the compound is fluid. Typical units given include Kelvin and degree Celsius. It is
+often found for organic chemicals as part of their unique characterization. Check out
+<a href="https://www.wikidata.org/wiki/Property:P2101#P1855">these examples</a>.
+
+<table class="table table-hover" id="missing-melting-points"></table>
+
+<h2 id="h1">Missing Mass</h2>
+
+<table class="table table-hover" id="missing-mass"></table>
+
+<h2 id="h1">Missing Solubility</h2>
+
+The <a href="https://www.wikidata.org/wiki/Property:P2177">solubility</a>
+of a compound is always in some solvent and depending on the temperature. The
+latter two are therefore commonly given as qualifiers, as for this
+<a href="https://www.wikidata.org/wiki/Property:P2177#P1855">example</a>.
+
+<table class="table table-hover" id="missing-solubility"></table>
+
+<h2 id="h1">Missing Enthalpy Of Formation</h2>
+
+The <a href="https://www.wikidata.org/wiki/Property:P3078">enthalpy of formation</a>
+is a measure of the stability of the compound.
+
+<table class="table table-hover" id="missing-enthalpy-of-formation"></table>
+
+<h2 id="h1">Missing Vapor Pressure</h2>
+
+<table class="table table-hover" id="missing-vapor-pressure"></table>
+
+<h2 id="h1">Missing Molar Entropy</h2>
+
+<table class="table table-hover" id="missing-molar-entropy"></table>
+
+<h2 id="h1">Missing Density</h2>
+
+<table class="table table-hover" id="missing-density"></table>
+
+<h2 id="h1">Missing Chemical Structure</h2>
+
+<table class="table table-hover" id="missing-chemical-structure"></table>
+
+<h2 id="h1">Missing Flash Point</h2>
+
+<table class="table table-hover" id="missing-flash-point"></table>
+
+<h2 id="h1">Missing pKa</h2>
+
+<table class="table table-hover" id="missing-pKa"></table>
+
+<h2 id="h1">Missing Combustion Enthalpy</h2>
+
+<table class="table table-hover" id="missing-combustion-enthalpy"></table>
+
+<h2 id="h1">Missing IDLH</h2>
+
+The <a href="https://www.wikidata.org/wiki/Property:P2129">Immediately Dangerous to Life and Health value</a>
+(IDLH) is an indication provided by the U.S.A. CDC about the danger of the compound.
```

Comment:
Opportunity to link to a CDC profile, perhaps https://tools.wmflabs.org/scholia/organization/Q583725 .

Reply:
Added